components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/account-settings-modal.tsx

```diff
@@ -274,52 +276,79 @@ export const AccountSettingsModal = () => {
           <DepositModal selectedAccount={selectedAccount} />
         )}
         {accountModalType === 'edit' && (
-          <>
+          <EditWrapper>
             <Input
               value={accountName}
               placeholder={getLocale('braveWalletAddAccountPlaceholder')}
-              onChange={handleAccountNameChanged}
+              onInput={handleAccountNameChanged}
               onKeyDown={handleKeyDown}
-            />
+              showErrors={showNameInputErrors}
+              maxlength={30}
```

Comment:
nit: BraveWallet.ACCOUNT_NAME_MAX_CHARACTER_LENGTH

Reply:
Thanks. I removed this prop value since the Nala component wasn't applying it
[a3e6c65](https://github.com/brave/brave-core/pull/22907/commits/a3e6c65516b717f2285997367093170880622356)